janus_server/src/aggregator.rs

```diff
@@ -24,39 +193,130 @@ fn aggregator_filter(task_id: TaskId) -> BoxedFilter<(impl Reply,)> {
                 "max-age=86400",
             )
         })
-        .with(trace::named("hpke_config"))
-        .boxed()
+        .with(trace::named("hpke_config"));
+
+    let upload_endpoint = warp::path("upload")
+        .and(warp::post())
+        .and(with_shared_value(aggregator))
+        .and(with_decoded_message())
+        .and_then(|aggregator: Aggregator<C>, report: Report| async move {
```

Comment:
Yup, I left myself a task to file an issue about rejection recovery. I have done this before in ppm-prototype but (1) I'm not sure what I did there is the best way to go about it and (2) it would have been a bunch more code in an already-large PR.